queries/python/highlights.scm

```diff
@@ -224,6 +224,7 @@
 ] @keyword.return
 (yield "from" @keyword.return)
 
+(future_import_statement "from" @include "__future__" @variable)
```

Comment:
probably also constant built-in for dunder variable to be consistent with this rule
```
((identifier) @constant.builtin
 (#lua-match? @constant.builtin "^__[a-zA-Z0-9_]*__$"))
```

Reply:
anyone still using Python2 :man_shrugging:

Reply:
Done!